### PR TITLE
feat: drop internal ingress

### PR DESCRIPTION
**Pull request by wood-push-melon**
This pull request aims to drop the internal ingress.

### CODE REVIEW

File: tests/integration/test_charm.py [hunk 1]
```diff
@@ -107,13 +107,14 @@ async def test_build_and_deploy(ops_test: OpsTest, local_charm: Path) -> None:
...
     await ops_test.model.wait_for_idle(
-        apps=[HYDRA_APP, DB_APP, PUBLIC_INGRESS_APP, INTERNAL_INGRESS_APP],
+        apps=[HYDRA_APP, DB_APP, PUBLIC_INGRESS_APP, ADMIN_INGRESS_APP],
         raise_on_blocked=False,
         status="active",
         timeout=5 * 60,
     )
 
 
+@pytest.mark.skip
```
Comment (pik4ez-canonical) [inline tests/integration/test_charm.py:117]: What is the reason to skip these tests?

**Review by pik4ez-canonical — CHANGES_REQUESTED** [review]
I'm curious about a bunch of skipped tests.

File: src/charm.py [hunk 2]
```diff
@@ -187,28 +188,10 @@ def __init__(self, *args: Any) -> None:
             self._on_database_integration_broken,
         )
 
-        # admin ingress
-        self.framework.observe(self.admin_ingress.on.ready, self._on_admin_ingress_ready)
```
Comment (natalian98) [inline src/charm.py:191]: Why is this removed? Don't we still need to observe the admin ingress-related events like we continue for the public one?

File: tests/integration/test_charm.py (hunk 1 above)
Comment (wood-push-melon) [reply tests/integration/test_charm.py:117]: ah, forgot to remove those when I did debugging. Let me remove those.

File: src/charm.py (hunk 2 above)
Comment (wood-push-melon) [reply src/charm.py:191]: this is because we don't do anything for admin-ingress integration joining and breaking.

**Review by pik4ez-canonical — APPROVED** [review]

**Review by BarcoMasile — APPROVED** [review]

File: src/charm.py (hunk 2 above)
Comment (wood-push-melon) [reply src/charm.py:191]: what I understand is that:

- We integrate with an admin ingress to expose admin API endpoints if we would like some client (outside the k8s cluster) to query
- For integration `hydra_endpoints`, we only pass the k8s svc addresses which are fixed values and for internal-cluster usage.

So admin ingress integration joining/departing does not change anything.

Let me know if I am wrong (we can chat a bit tomorrow)

Comment (natalian98) [reply src/charm.py:191]: Ok, I missed this part:

> For integration hydra_endpoints, we only pass the k8s svc addresses which are fixed values and for internal-cluster usage.

It's fine to remove it then.

**Review by natalian98 — APPROVED** [review]
